diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/BlockStructureBraces.kt

```diff
@@ -43,6 +44,10 @@ import org.jetbrains.kotlin.psi.KtTryExpression
 
 class BlockStructureBraces : Rule("block-structure") {
 
+    companion object{
+        val emptyBlockList = listOf(LBRACE, WHITE_SPACE, RBRACE)
```

Comment:
Please import this constant from utils instead of creating it in multiple places.